README.md

```diff
@@ -23,18 +26,25 @@ Paste the following Terraform sample module into file clickhouse-eks.tf in a new
 ```hcl
 provider "aws" {
   # https://registry.terraform.io/providers/hashicorp/aws/latest/docs
+  # region     = "us-east-1"
```

Comment:
sure. If you are TF guy. I actually  first deleted aws section completely, because it is not needed. But later I understood that it is for documentation and that there are some useful parameters over there.

Reply:
It's actually better to remove it. 
Something else to consider, this README is not the blueprint. This is meant for people with Terraform experience which nows what a module is. It's better to keep this as simple as possible and follow existing patterns. We then have the blueprint for detailed explanations